### ISSUE TITLE
Infinite Scroll with Experiment version `1903080058260`

### ISSUE THREAD

**Issue by leonalicious** [ampproject/amphtml#21403]
## What's the issue?

Briefly describe the bug.

## How do we reproduce the issue?
0. Turn on AMP Dev Channel
1. Visit [https://m.aliexpress.com/wholesale/dress.html?spm=a2g0n.home-amp.0.0&categoryId=200003482&channel=direct](https://m.aliexpress.com/wholesale/dress.html?spm=a2g0n.home-amp.0.0&categoryId=200003482&channel=direct)
2. Even encounter issue while I am refreshing on the page
![image](https://user-images.githubusercontent.com/4065175/54303891-f4a56300-45fe-11e9-827b-35ff064f2e82.png)
Will encounter issue below while triggering the infinite scroll
![image](https://user-images.githubusercontent.com/4065175/54303969-228aa780-45ff-11e9-9c5f-f53c76d6a96a.png)

FYI, we have increased our traffic to 100%, so your guys will hit into Infinite scroll version of Search List easily.

## What browsers are affected?

Chrome

## Which AMP version is affected?
`version 1903080058260` (Experiment version)

/cc: @cathyxz

**Comment by cathyxz** [comment]
According to my understanding, there are two separate issues at hand here: 
1. If the placeholder is larger than the list height or a viewport resize occurs sufficiently early, there is a race condition that results in the JS console error shown above. This is an implementation bug and should be fixed by https://github.com/ampproject/amphtml/pull/21390, which includes a reliable repro for this bug in the testing code. 
2. If you scroll the site attached in the issue description fast enough, you will visually see the `load-more-button` flicker before turning into the loader. This is the intended behavior of our current implementation--the see more button is shown by default and toggled to the loader on triggering a load. This behavior is reasonable for the `manual` load-more case, but I think it's a UX bug in the automatic case, since having a flickering button is not actually a good UX experience if it's going to load automatically and go away anyway. 

For 2, I'm proposing that in the automatic load more case (i.e. infinite scroll), that we never show the `see-more-button` (except to screen readers), since we have a separate `load-more-failed` case and the `see-more-button` will never actually be actionable if you make use of scrolling.

**Comment by cathyxz** [comment]
Got a reliable repro for the JS console issue here: https://m.aliexpress.com/category/200003378/men%27s-costumes.html?categoryName=Men%27s+Costumes&spm=a2g0n.category-amp.leafcat.Men%27s%20Costumes

```
Powered by AMP ⚡ HTML – Version 1903141753530 https://m.aliexpress.com/category/200003378/men%27s-costumes.html?categoryName=Men%27s+Costumes&spm=a2g0n.category-amp.leafcat.Men%27s%20Costumes
```

**Comment by cathyxz** [comment]
I think we separately solved the `getMoreEndElement` issue in https://github.com/ampproject/amphtml/pull/21390 already by fixing the race condition on accessing the loadMoreEndElement before initializing it. Since the remaining issue here is cosmetic and not blocking functionality, I'm downgrading to P2.
